File: SMarketPlace/SMarketPlace.Web/Modules/ServerTypes/Ecommerce/ActualizarPedidoForm.ts
import { LookupEditor, StringEditor, DateEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface ActualizarPedidoForm {
    IdPedidoEcommerce: LookupEditor;
    EstadoPedidoInternet: StringEditor;
    Estado: StringEditor;
    DtRegistroUsr: DateEditor;
    DtCambio: DateEditor;
}

export class ActualizarPedidoForm extends PrefixedContext {
    static formKey = 'Ecommerce.ActualizarPedido';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ActualizarPedidoForm.init)  {
            ActualizarPedidoForm.init = true;

            var w0 = LookupEditor;
            var w1 = StringEditor;
            var w2 = DateEditor;

            initFormType(ActualizarPedidoForm, [
                'IdPedidoEcommerce', w0,
                'EstadoPedidoInternet', w1,
                'Estado', w1,
                'DtRegistroUsr', w2,
                'DtCambio', w2
            ]);
        }
    }
}

File: SMarketPlace/SMarketPlace.Web/Modules/Ecommerce/ActualizarPedido/ActualizarPedidoForm.cs
using Serenity.ComponentModel;
using System;

namespace SMarketPlace.Ecommerce.Forms;

[FormScript("Ecommerce.ActualizarPedido")]
[BasedOnRow(typeof(ActualizarPedidoRow), CheckNames = true)]
public class ActualizarPedidoForm
{
    public string IdPedidoEcommerce { get; set; }
    public string EstadoPedidoInternet { get; set; }
    public string Estado { get; set; }
    public DateTime DtRegistroUsr { get; set; }
    public DateTime DtCambio { get; set; }
}